sdk/eventhub/event-hubs/samples/browserSample/README.md

```diff
@@ -0,0 +1,81 @@
+---
+page_type: sample
+languages:
+  - javascript
+products:
+  - azure
+  - azure-event-hubs
+urlFragment: event-hubs-javascript
```

Comment:
`urlFragment` has to be unique for every sample package, so it needs to be something like `event-hubs-javascript-browser`.

Reply:
Updated!